src/core/Utils.re

```diff
@@ -1,2 +1,3 @@
 let foi = float_of_int;
-let iof = int_of_float;
\ No newline at end of file
+let iof = int_of_float;
+let sof = Js.Float.toString;
```

Comment:
Reason does have `string_of_float` but it apparently is quite poor at producing accurate strings, so the compiler recommends `Js.Float.toString`.